oauth2client/crypt.py

```diff
@@ -57,10 +57,13 @@ def _TryOpenSslImport():
 
   """
   try:
-    _ = imp.find_module('OpenSSL')
+    _, _package_dir, _ = imp.find_module('OpenSSL')
+    if not (os.path.isfile(os.path.join(_package_dir, 'crypto.py')) or
+            os.path.isfile(os.path.join(_package_dir, 'crypto.so'))):
+      raise ImportError('No module named OpenSSL.crypto')
```

Comment:
i was letting those folks fall through to explicit import -- but you're right, it's easy to catch here.